README.md

```diff
@@ -1 +1,35 @@
-# Portfolio
\ No newline at end of file
+# Portfolio
+
+This project is about building a Portfolio using the mobile-first approach. This is the first step of the whole project for my portfolio. According to project requirements, we must follow one of the designs from provided templates to design the website and I have decided to follow the "Template 2" design and look from Figma.
+
+
+## Build with
+
+- HTML
+- CSS
+
+## Live Demo
+https://github.com/developertariq/Portfolio
+
+## Getting Started
+
+### Prerequisites
+### Setup
+### Install
+### Usage
+### Deployment
+
+## Authors
+
+#### Md Tariq-ul Islam
+- GitHub: 
+- LinkedIn:
+- Twitter:
+
+## Contributing
+
+## Show your support
+
+## Acknowledgments
+
+## License
```

Comment:
- [ ] As the previous reviewer asked, kindly fill out these sections of the README file with information about your project and you to have a professional looking README file ✨